demos/GMW_demo/1-out-of-n/ot.js

```diff
@@ -0,0 +1,153 @@
+module.exports = function (io, sodium) {
+  const util = require('./util.js')(sodium);
+  const crypto = require('./crypto.js')(sodium, util);
+
+  // 1-out-of-2 OT sending
+  const send_from_2 = function (X1, X2, op_id, jiff,session_id) {
+    op_id = op_id + ':1in2ot';
+    var get = io.get.bind(null, op_id,jiff,session_id);
+    var give = io.give.bind(null, op_id, jiff,session_id);
+
+    const a = sodium.crypto_core_ristretto255_scalar_random();
+    const A = sodium.crypto_scalarmult_ristretto255_base(a);
+
+    give('A', util.to_str(A));
+    get('B').then(function (B_str) {
+      const B = util.from_str(B_str);
+
+      var k0 = sodium.crypto_scalarmult_ristretto255(a, B);
+      var k1 = sodium.crypto_scalarmult_ristretto255(a, sodium.crypto_core_ristretto255_sub(B, A));
+
+      k0 = sodium.crypto_generichash(32, k0);
+      k1 = sodium.crypto_generichash(32, k1);
+
+      const e0 = crypto.encrypt_generic(X1, k0);
+      const e1 = crypto.encrypt_generic(X2, k1);
+      give('e', util.to_str([e0, e1]));
+    });
+  };
+
+  // 1-out-of-2 OT receiving
+  const receive_from_2 = function (c, op_id, jiff,session_id) {
+    op_id = op_id + ':1in2ot';
```

Comment:
it looks like these are the same as the original 1-out-of-n library, so should be okay when we use it as an npm package (need to deal with jiff_instance parameter as well but I think that will be okay)